labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+
+### Using the UI
+
+* Log in to the Vendor Portal using your email and password.
+* On the navigation menu, select **Releases**
+
+    <p align="center"><img src="img/releases.png" alt="vendor-ui" width="1000" margin=auto/></img></p>
+
+* Click on **Create Release**
+
+    <p align="center"><img src="img/create-release.png" alt="create-release" width="1000" margin=auto/></img></p>
+
+* We will not use the default nginx app, so let's delete the following files:
+    * example-configmap.yaml
+    * example-deployment.yaml
+    * example-service.yaml
+
+    <p align="center"><img src="img/delete-files.png" alt="files-to-delete" width="1000" margin=auto/></img></p>
+
+* Now we are ready to add the Chart! Open a window and browse to the directory where the Helm Chart we created is located. From your terminal on macos, you can use `open .` to quickly get a Finder window open.
+* Drag and Drop it where the Replicated Release Files navigator as shown below
+    <p align="center"><img src="img/dragchart.png" alt="drag and drop chart" width="1000" margin=auto/></img></p>
+
+* You will be prompted to select which method to use when deploying the Helm Chart:
+   
+    <p align="center"><img src="img/helm-opts.png" alt="helm options" width="600" margin=auto/></img></p>
+
+   * **Native Helm** - This should be the default option and should be the one selected. With this option, App Manager will use Helm to deploy the Helm Chart in the customer's environment.
+   * **Replicated** - This option will eventually be depecrated and should not be selected. With this option, App Manager will first generate Kubernetes Manifests from the Helm Chart (i.e., run `helm template`) before they are deployed to the cluster as manifests.
+
+* Note how a new file is automatically created by Replicated that includes details about the chart we just added.
+
+<p align="center"><img src="img/helm-file.png" alt="helm file" width="600" margin=auto/></img></p>
+
+As a last step, save the release we've created 
+<p align="center"><img src="img/save release.png" alt="helm file" width="600" margin=auto/></img></p>
+
+### Using the CLI
+
+For the CLI, you should have already completed the [Hello World](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md) Lab, as it covers how to set up your local Dev environment with the Replicated CLI.
+
+Make sure to update your environment variables to interact with this application. See [Get Started -> Steps 1 and 2](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md)
+
+
+`REPLICATED_APP` should be set to the app slug from the Settings page. You should have received your App Name
```

Comment:
I think this might be confusing. Whether they received it or not, they don't actually need it to navigate to settings and get the slug, right?

```suggestion
`REPLICATED_APP` should be set to the app slug from the Settings page.
```

Reply:
And they never really receive it. Since we have rbac, this part should be removed from all labs instructions.